src/early_multicellular_stage/editor/CellBodyPlanEditorComponent.cs

```diff
@@ -1002,7 +1002,7 @@ private void ShowCellTypeInModelHolder(SceneDisplayer modelHolder, CellType cell
         microbe.ApplySpecies(newSpecies);
 
         // Apply placeholder scale if doesn't have a scale
-        if (microbe.Membrane.Scale == Vector3.One)
+        if (microbe.Membrane?.Scale == Vector3.One)
```

Comment:
Is this really needed?

Reply:
Yes, membrane can be null.

Reply:
How? There's ApplySpecies a couple of lines above this, I'm very sure that should not let any null membranes through... or has that been changed? If the situation has not been changed I'd prefer an explicit exception throw in an error case here.

Reply:
Yes I made Membrane a nullable, I actually put exceptions in a lot of other places so will put here too.